js_machine_front/src/scenes/news/news.tsx

```diff
@@ -1,18 +1,48 @@
-import React, { memo } from 'react';
-import Background from './images/news.jpg';
-
-const sectionStyle = {
-  height: '100vh',
-  backgroundImage: `url(${Background})`,
-  backgroundRepeat: 'no-repeat',
-  backgroundSize: 'cover',
-  backgroundPosition: 'center',
-};
-
-export const News: React.FC = memo(() => {
-  return (
-    <div style={ sectionStyle }>
-      <div className="body"/>
-    </div>
-  );
-});
+import React, { PureComponent } from 'react';
+import { DigestView } from 'components/DigestView/DigestView';
+
+const sectionStyle = {
+  backgroundColor: 'grey',
+  paddingTop: 100,
+};
+
+interface State {
+  openDigest: boolean;
+}
+
+export class News extends PureComponent<{}, State> {
+
+  constructor(props: {}) {
```

Comment:
please remove constructor, you can define you state with initial values as a public property